server/services/v1/query_runner.go

```diff
@@ -1203,6 +1212,19 @@ func (runner *PublishQueryRunner) publish(ctx context.Context, tx transaction.Tx
 	return ts, allKeys, err
 }
 
+func partFromFields(fields []*schema.Field, message []byte) uint16 {
+	hash := fnv.New32()
+
+	for _, field := range fields {
+		val, _, _, _ := jsonparser.Get(message, field.FieldName)
```

Comment:
Do we need it here? Also, best if the field is passed from the collection. Basically, once we create the collection we already pass the fields. If these fields have the partition key attribute set then we just need to rely on that attribute.